nibabel/streamlines/header.py

```diff
@@ -0,0 +1,19 @@
+
+class Field:
```

Comment:
Or you could make a class for the header information, so the header would be an instance of this class, and where the class might have value checking properties for the attributes...